sw/otbn/code-snippets/p256.s

```diff
@@ -1919,13 +1727,53 @@ p256_scalarmult:
   /* store result (affine coordinates) in dmem
      dmem[x21] = dmem[dptr_x] <= x_a = w11
      dmem[x22] = dmem[dptr_y] <= y_a = w12 */
-  bn.sid    x11++, 0(x21)
-  bn.sid    x11++, 0(x22)
+  li        x2, 11
+  bn.sid    x2++, 0(x21)
+  bn.sid    x2, 0(x22)
 
   ret
 
 .section .data
 
+/* pointer to k (dptr_k) */
+.globl dptr_k
```

Comment:
add a `.balign 4` before this section starts to ensure that pointers are 32b word-aligned.